pkg/resources/fluentd/config.go

```diff
@@ -54,7 +54,6 @@ var fluentdInputTemplate = `
 <source>
     @type prometheus
     port {{ .Monitor.Port }}
-    metrics_path {{ .Monitor.Path }}
```

Comment:
But how do we notify users who accidentally configured this to something different than `/metrics` in the default case?

I see a few options here:
- configure the annotation and the fluentd config differently (leave the fluentd config empty) and update the template to configure the path only if it's set.
- update the template to avoid setting the path if it's explicitly `/metrics or /aggregated_metrics`

Reply:
I did the first option. Let me know what you think about it.